SI/Sem.02/Pract.02/Task02.cpp

```diff
@@ -1,33 +1,50 @@
-﻿#include <iostream>
+// task02.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
 
-int main() {
-	char inputCharacter;
+#include <iostream>
+using namespace std;
 
-	std::cin >> inputCharacter;
-
-	if ((inputCharacter >= 'A' && inputCharacter <= 'Z') || (inputCharacter >= 'a' && inputCharacter <= 'z')) {
-		switch (inputCharacter) {
-			case 'A':
-			case 'E':
-			case 'I':
-			case 'O':
-			case 'U':
-			case 'Y':
-			case 'a':
-			case 'e':
-			case 'i':
-			case 'o':
-			case 'u':
-			case 'y':
-				std::cout << "Entered letter is a vowel.\n";
-				break;
-			default:
-				std::cout << "Entered letter is not a vowel.\n";
-				break;
-		}
-	} else {
-		std::cout << "The symbol is not part of the latin alphabet." << std::endl;
+int main()
+{
+	char symbol;
+	cout << "Enter a symbol:";
+	cin >> symbol;
+	if('A'<= symbol && symbol <='Z'|| 'a'<= symbol && symbol <='z')
+	{
+		cout << "This symbol is from the Latin alphabet."<< endl;
+	}
+	else
+	{ 
+		cout << "This symbol is not from the Latin alphabet" << endl; 
+	}
+	switch(symbol)
+	{
+	case'A':cout << "vowel";
```

Comment:
You can check the solution that is in Pract.02/task02.cpp.
The problem here is we are repeating a lot of our code unnecessary.